### PR TITLE
[C-1650] Finalize solana-phone integration

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Finalizes solana-phone flow
Addresses some issues with wallet-connect on android

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-wallet-connect-android

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-wallet-connect-android

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-wallet-connect-android

File: packages/mobile/src/screens/audio-screen/AudioScreen.tsx [hunk 1]
```diff
@@ -185,6 +186,12 @@ export const AudioScreen = () => {
     dispatch(setVisibility({ modal: 'AudioBreakdown', visible: true }))
   }, [dispatch])
 
+  useEffect(() => {
```
Comment (raymondjacobson) [inline packages/mobile/src/screens/audio-screen/AudioScreen.tsx:189]: 🚒

File: packages/web/src/common/store/pages/token-dashboard/checkIsNewWallet.ts [hunk 2]
```diff
@@ -15,16 +15,23 @@ export function* checkIsNewWallet(walletAddress: string, chain: Chain) {
     getAssociatedWallets
   )
 
+  console.log('geting here?')
```
Comment (raymondjacobson) [inline packages/web/src/common/store/pages/token-dashboard/checkIsNewWallet.ts:18]: got here

**Review by dylanjeffers — APPROVED** [review]

**Pull request merged**